localization/fr.ftl

```diff
@@ -0,0 +1,178 @@
+folder-tools-desc = Ajoute des boutons en mode revision de liste pour aider à gérer les dossiers.
+folder-tools-dump = Décharger
+folder-tools-merge = Fusionner
+folder-tools-enclose = Enfermer
+
+## Video Creator
+video-creator-name = Créateur de Vidéos
+video-creator-desc = Créer des vidéos et des GIFs de vos graphiques en fonction d'actions ou de curseurs.
+video-creator-menu = Menu du Créateur de Vidéos
+video-creator-to = à
+video-creator-step = , étape
+video-creator-prev-action = Précédent
+video-creator-next-action = Suivant
+video-creator-size = Taille:
+video-creator-step-count = Nombe d'étape:
+video-creator-target-same-pixel-ratio = Cibler le même ratio de pixels
+video-creator-target-tooltip = Ajuste la mise à l'échelle de la largeur de ligne, de la taille des points, de la taille de l'étiquette, etc.
+video-creator-ffmpeg-loading = Chargement de FFmpeg...
+video-creator-ffmpeg-fail = Si cela ne fonctionne pas dans quelque instants, essayez de recharger la page ou de signaler ce bogue aux développeurs de DesModder.
+video-creator-exporting = Rendition en cours...
+video-creator-cancel-capture = Annuler
+video-creator-cancel-export = Annuler
+video-creator-capture = Enregistrer
+video-creator-preview = Avant-première
+video-creator-delete-all = Tout Suprimer
+video-creator-filename-placeholder = définir le nom de fichier
+video-creator-export = Créer
+video-creator-export-ff = Créer (Avertissement : actuellement peu fiable/lent dans Firefox)
+video-creator-export-as = Créer comme { $fileType }
+video-creator-fps = Images par seconde:
+video-creator-method-once = unique
+video-creator-method-slider = curseurs
+video-creator-method-action = action
+
+## Shift+Enter Newline
+shift-enter-newline-name = Maj+Entrée Nouvelle Ligne
+shift-enter-newline-desc = Utilisez Maj+Entrée pour taper des retours à la ligne dans les notes et les titres des images/dossiers.
+
+## Wakatime
+wakatime-name = WakaTime
+wakatime-desc = Suivez votre activité sur Desmos sur WakaTime.com
+wakatime-opt-secretKey-name = Clef Secrète
+wakatime-opt-secretKey-desc = Clé API utilisée pour les serveurs WakaTime
+wakatime-opt-splitProjects-name = Diviser les projets par graphique
+wakatime-opt-splitProjects-desc = Stockez chaque graphique comme son propre projet au lieu des branches d'un projet unifié
+wakatime-opt-projectName-name = Nom du projet
+wakatime-opt-projectName-desc = Visible from WakaTime, and shared for all Desmos projects
+
+## Performance Display
+performance-info-name = Performance Display
+performance-info-desc = Affiche des informations sur les performances du graphique actuel.
+performance-info-refresh-graph = Actualiser le graphique
+performance-info-refresh-graph-tooltip = Actualisez le graphique pour mesurer le temps de chargement initial
+performance-info-sticky-tooltip = Garder le Menu Ouvert
+performance-info-time-in-worker = Time In Worker
+performance-info-compiling = Compilation
+performance-info-rendering = Rendering Rendition
```

Comment:
Is this really two words? Google translate directly translates "rendering" as "le rendu"